README

```diff
@@ -780,17557 +783,468 @@ export PATH CPPFLAGS LDFLAGS
-   settings panel.)
-
-
-   Here are some features that will appear in the 0.9.13 release:
-
-     * Improved support for non-X11 touchscreen devices (e.g. handheld or
-       cell phone) via Linux uinput input injection. Additional tuning
-       parameters are added. TSLIB touchscreen calibration is supported.
-       Tested on Qtmoko Neo Freerunner. A tool, misc/uinput.pl, is
-       provided to diagnose uinput behavior on new devices. The env.
-       vars. X11VNC_UINPUT_BUS and X11VNC_UINPUT_VERSION are available if
-       leaving them unset does not work.
-     * The Linux uinput non-X11 input injection can now be bypassed:
-       events can be directly written to the /dev/input/event devices
-       specified by the user (direct_abs=..., etc.) A -pipeinput input
-       injection helper script, misc/qt_tslib_inject.pl is provided as a
-       tweakable non-builtin direct input injection method.
-     * The list of new uinput parameters for the above two features is:
-       pressure, tslib_cal, touch_always, dragskip, btn_touch;
-       direct_rel, direct_abs, direct_btn, direct_key.
-     * The MacOSX native server can now use OpenGL for the screen
-       capture. In nearly all cases this is faster than the raw
-       framebuffer capture method. There are build and run time flags,
-       X11VNC_MACOSX_NO_DEPRECATED, etc. to disable use of deprecated
-       input injection and screen access interfaces. Cursor shape now
-       works for 64bit binaries.
-     * The included SSL enabled Java VNC Viewers now handle Mouse Wheel
-       events.
-     * miscellaneous new features and changes:
-     * In -reflect mode, the libvncclient connection can now have the
-       pixel format modified via the environment variables
-       X11VNC_REFLECT_bitsPerSample, X11VNC_REFLECT_samplesPerPixel, and
-       X11VNC_REFLECT_bytesPerPixel
-     * In -create mode the following environment variables are added to
-       fine tune the behavior: FIND_DISPLAY_NO_LSOF: do not use lsof(1)
-       to try to determine the Linux VT, FIND_DISPLAY_NO_VT_FIND: do not
-       try to determine the Linux VT at all, X11VNC_CREATE_LC_ALL_C_OK:
-       do not bother undoing the setting LC_ALL=C that the create_display
-       script sets. The performance of the -create script has been
-       improved for large installations (100's of user sessions on one
-       machine.)
-     * In -unixpw mode, one can now Tab from login: to Password.
-     * An environment variable, X11VNC_SB_FACTOR, allows one to scale the
-       -sb screenblank sleep time from the default 2 secs.
-     * An experimental option -unixsock is available for testing. Note,
-       however, that it requires a manual change to
-       libvncserver/rfbserver.c for it to work.
-     * Documented that -grabkbd is no longer working with some/most
-       window managers (it can prevent resizing and menu posting.)
-
-
-   Here are some features that appeared in the 0.9.12 release (Sep/2010):
```

Comment:
those should be synced with NEWS probably

Reply:
I'll add a top-level 'News' section between 'Annoucements' & 'FAQs' & drop a link to the NEWS file there.